### DISCUSSION TITLE
Disabling featured content in a component

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#618] [Ideas]
When creating news sites, sometimes there are some moments to make some materials favorites in order to display them in modules. But it, in turn, is shown on the main page if there is a Component display. 
![IMG_20240327_155725_188](https://github.com/templaza/astroid-framework/assets/40716604/883f057c-7a09-4f5f-b8ca-80f734e9932a)

Is it possible in the Component settings to not show selected materials on the main one? than to create a copy of the template style, on a disabled component, and show it on the rest.

**Comment by rinka88** [comment]
@sonvnn 
Can you add a menu Assignment here? To manage these elements in the main page

**Comment by sonvnn** [comment]
@rinka88 I added Menu Assignment to Component and Message element here https://github.com/templaza/astroid-framework/commit/fa4b9611350042d04bf3fc0c41a5dab3f76c29c3

You can download v3-branch and test it.

Thanks & Best Regards,
Sonny

**Comment by rinka88** [comment]
Tested it. It works perfectly

**Comment by rinka88** [comment]
@sonvnn After installing RC-2, the management tab of the menu is gone

**Reply by sonvnn** [reply]
Please send me a screenshot of issue.

Thank you

**Comment by rinka88** [comment]
@sonvnn 
![IMG_20240418_174620_454](https://github.com/templaza/astroid-framework/assets/40716604/d50e069f-1fec-424e-a200-6c4eb6c15f9c)
in the first screenshot rc-2 

in the second screenshot rc1
![IMG_20240418_174620_812](https://github.com/templaza/astroid-framework/assets/40716604/17d932be-eeb5-41b0-8d45-10e4159fff11)
Joomla 5.1

**Reply by sonvnn** [reply]
Thank you I will check it tomorrow :)

**Reply by sonvnn** [reply]
@rinka88 I made a mistake. I updated v3.0.14-rc2 to latest here https://github.com/templaza/astroid-framework/releases/tag/v3.0.14-rc2

Please help me download and retest again.

Best Regards,
Sonny

**Reply by rinka88** [reply]
It's working now
